haskell/gen_ghc_bindist.py

```diff
@@ -18,6 +18,7 @@
 # `ignore_suffixes` is the suffix of files to ignore
 VERSIONS = [
     { "version": "8.6.4" },
+    { "version": "8.6.5" },
```

Comment:
My OCD says this should move one line further up. 😃

Reply:
Fixed :smile: